src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -413,8 +417,12 @@ public void shouldNotSendPatronNoticeWhenCheckInNoticeIsDefinedAndCheckInDoesNot
     assertThat("Response should not include a loan",
       checkInResponse.getJson().containsKey("loan"), is(false));
 
-    List<JsonObject> sentNotices = patronNoticesClient.getAll();
-    MatcherAssert.assertThat("Notice shouldn't be sent second time",
-      sentNotices.size(), Matchers.lessThan(2));
+    Awaitility.await()
+      .timeout(1, TimeUnit.SECONDS)
+      .untilAsserted(() -> {
+        List<JsonObject> sentNotices = patronNoticesClient.getAll();
+        MatcherAssert.assertThat("Notice shouldn't be sent second time",
+          sentNotices.size(), Matchers.lessThan(2));
```

Comment:
If we only expect a single notice, why isn’t this specifically one notice?

Reply:
I haven't noticed that sent notices are not cleared after the test. Fixed this, and change asset to _Matchers.empty()_